testsuite/smoke-test/src/storage.rs

```diff
@@ -349,6 +349,7 @@ pub(crate) fn db_backup(
     get_backup_storage_state(&bin_path, metadata_cache_path2.path(), backup_path.path()).unwrap();
 
     // spawn the backup coordinator
+    #[allow(clippy::zombie_processes)]
```

Comment:
Note @msmouse I didn't want to change this behavior, but you may want to look at improving this test